dspace/solr/search/conf/schema.xml

```diff
@@ -281,6 +281,9 @@
     <field name="SolrIndexer.lastIndexed" type="date" indexed="true" stored="true" default="NOW" multiValued="false" omitNorms="true" />
     <field name="lastModified" type="date" indexed="true" stored="true" default="NOW" multiValued="false" omitNorms="true" />
 
+     <!-- The database status of the current item -->
+     <field name="status" type="string" indexed="true" stored="true" omitNorms="true" docValues="true" />
```

Comment:
We might want to consider renaming this, as Solr returns it's own `status` field for the response code (e.g. `status: 0` = no errors).  While it looks Solr might let us add this to the schema, it could be confusing for other developers reading this code later.  We might consider renaming to be `database_status`

Reply:
Renamed this to database_status and applied this to static property ItemIndexFactoryImpl.STATUS_FIELD, which is referred to everywhere in the code, so this was the only change needed.